run/FuzzyDarkMatter/init.py

```diff
@@ -0,0 +1,84 @@
+import h5py as h5
+import numpy as np
+from scipy import *
+# Read Baryon density
+
+rank = 3
+dim  = 64
+
+attr = {'Component_Rank':1, 'Component_Size':dim**rank, 'Dimensions':[dim]*rank, 'Rank':rank, 'TopGridDims':[dim]*rank, 'TopGridEnd':[dim]*rank, 'TopGridStart':[0]*rank}
+
+# renormalize so mean is 1
+dens = 1e-6*np.ones([dim]*rank)
+xdim,ydim,zdim = dens.shape
+
+for i in range(xdim):
```

Comment:
Correct -- I added a comment but I'm not sure it's really issue level since it is only 64^3 test.  Practically speaking, I'm not sure I see the need to fix.